CHANGES.md

```diff
@@ -2,6 +2,8 @@
 
 ### 1.113 - 2024-01-02
 
+- Added a new Sandcastle example, Bing Maps Labels Only, which demonstrates using the Cesium ion imagery asset of the same name.
```

Comment:
Typically, we do not document Sandcastle changes in `CHANGES.md`. We do mention them in release posts and will do so there.